examples/storybooks/generic.scss

```diff
@@ -1,17 +1,31 @@
-.sourceLink {
+.sourceLink,
+.sandboxLink {
   position: fixed;
   top: 0;
   right: 0;
   padding: 130px 50px 5px 50px;
   font: 10px helvetica, sans-serif;
   display: inline-block;
-  background: rgba(12, 35, 194, 0.7);
+  background: rgb(12, 35, 194);
   color: #fff;
   text-decoration: none;
   transform: translate(50%, -50%) rotateZ(45deg);
   transition: background 100ms;
 
   &:hover:not(:active) {
-    background: rgba(102, 135, 244, 0.9);
+    background: rgb(102, 135, 244);
+  }
+}
+
+.sandboxLink {
+  top: 30px;
+  right: 30px;
+  background: rgb(12, 194, 68);
+  padding: 130px 100px 5px 100px;
+  border: none;
+  cursor: pointer;
+  outline: inherit !important;
```

Comment:
What is the outline rule for here?

Reply:
I did that because now it is rendered as a button and looks really weird when you click it and wait for the redirect. But I will change it to `outline: none;` which makes it clearer